projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -566,6 +573,30 @@ public Response getStandings(
 					// we got the internal contest; pass it to the DefaultScoringAlgorithm and get back updated standings
 					try {
 						Properties props = internalContest.getContestInformation().getScoringProperties();
+
+						xlog(logger, "debug 22 props "+props);
+						if (props != null) {
+						       Set<Object> set = props.keySet();
+
+						        String[] keys = (String[]) set.toArray(new String[set.size()]);
+
+						        Arrays.sort(keys);
+								xlog(logger, "debug 22   Scoring Properties, there are " + keys.length + " keys");
+
+						        for (String string : keys) {
+									xlog(logger, "debug 22      " + string + "='" + props.get(string) + "'");
+						        }
+						}
+						
+						props = DefaultScoringAlgorithm.getDefaultProperties();
+						xlog(logger, "debug 22  set Default Scoring Props for "+key);
+
+						
+						// TODO use key to find clientId then use clientId to find division runs for this client
+//						ClientId clientId = null;
+//						Run[] runs = ScoreboardUtilites.getRunsForUser(clientId, internalContest);
+						
+                        Run[] runs = internalContest.getRuns();
```

Comment:
What is the purpose of the above statement?  It defines a variable which (as far as i can see) never used.  Please remove, or explain why it's there.

Reply:
Hmm, the commented out section is how we'll filter the team's runs

Reply:
The getRuns is existing code, it seems to be a github diff thing, the getRuns is not new.